.rubocop.yml

```diff
@@ -6,7 +6,6 @@ AllCops:
     - 'db/migrate/*'
     - 'lib/templates/**/*'
     - 'node_modules/**/*'
-    - 'bin/**/*'
```

Comment:
not 100% sure about this but I do link the linter to help me see code optimization tips, even in these bin files